web-client/src/presenter/actions/TrialSession/setTimeStampAction.test.ts

```diff
@@ -0,0 +1,14 @@
+import { DateTime } from 'luxon';
+import { runAction } from '@web-client/presenter/test.cerebral';
+import { setTimeStampAction } from '@web-client/presenter/actions/TrialSession/setTimeStampAction';
+
+describe('setTimeStampAction', () => {
+  it('should set time stamp', async () => {
+    const propertyName = 'FetchedTrialSessions';
+    const result = await runAction(setTimeStampAction({ propertyName }), {});
+
+    const expectedDate = DateTime.now().setZone('America/New_York').toISODate();
```

Comment:
Can we just use the `formatNow` function (or some other function) in `DateHandler` rather than importing `luxon`? (That way we keep luxon in one spot.)